src/main/java/com/amazon/asksdk/cloudfoody/Restaurant.java

```diff
@@ -0,0 +1,102 @@
+package com.amazon.asksdk.cloudfoody;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class Restaurant extends OpenTable {
```

Comment:
Restaurant shouldn't extend OpenTable, because it is not a different version of "OpenTable", but a separate class.

Furthermore,  Restaurant class shouldn't have references to OpenTable methods. Please consider making Restaurant a POJO (Plain Old Java Object), and methods in OpenTable client returning instances of Restaurant. 

You could consider bringing a dependency on Jackson to map JSON responses into Restaurant instances (
http://tutorials.jenkov.com/java-json/jackson-objectmapper.html) 

https://github.com/FasterXML/jackson-core

Reply:
I made Restaurant a POJO.
I would like to use jackson to pam the JSON response to a restaurant class but there are many levels in the response.